lib/controller/python/controller/field.py

```diff
@@ -155,16 +156,28 @@ def setSFFloat(self, value: float):
         self.value = value
 
     def setSFVec2f(self, value: typing.List[float]):
-        self.value = value
+        if len(value) == 2:
+            self.value = value
+        else:
+            print("Error: setSFVec2f(): 'value' argument should have length 2.", file=sys.stderr)
```

Comment:
It doesn't sound like good English.
```suggestion
            print("Error: setSFVec2f(): 'value' argument should have a length of 2.", file=sys.stderr)
```
or
```suggestion
            print("Error: setSFVec2f(): length of 'value' argument should be 2.", file=sys.stderr)
```